.gitignore

```diff
@@ -14,3 +14,5 @@ keyfile.json
 
 tests/assets/integration/trail-path.yml
 tests/assets/integration/credentials.yml
+credentials.yml
+replica.yml
```

Comment:
These two files are used by the init/new command, so should still be included. We'd need to rename some things if we want to always exclude generic replica/credentials files.

Reply:
On my end files in templates folder were not affected, I will double check this.

Reply:
Yep, only the actual custom files are affected, templates are safe. Or did I misunderstood why we need that?